sharpy/generators/polaraeroforces.py

```diff
@@ -231,9 +241,34 @@ def generate(self, **params):
                         moment_s += moment_polar_lift
 
                     new_struct_forces[inode, 3:6] = c_bs.dot(moment_s)
+        if self.settings['write_induced_aoa']:
+            self.write_induced_aoa_of_each_node(ts, folder, list_aoa_induced)
 
         return new_struct_forces
     
+    def write_induced_aoa_of_each_node(self,ts, folder,list_aoa_induced):
+        '''
+        Writes induced aoa of each node to txt file for each timestep. 
+
+        Args:
+            ts (int): simulation timestep 
+            folder (str): output folder to which indced aoa files shall be exported
```

Comment:
*induced*

Reply:
Thanks Alvaro. I have changed the function in a later commit, resulting in removing the description including the typo anyway.